File: original_scripts/julia/matrix_multiplication/qwen3_8b.jl
function matrix_mult(A, B)
    m, n = size(A)
    n2, p = size(B)
    C = zeros(eltype(A), m, p)
    for i in 1:m
        for j in 1:p
            for k in 1:n
                C[i, j] += A[i, k] * B[k, j]
            end
        end
    end
    return C
end

File: original_scripts/python/matrix_multiplication/qwen3_8b.py
def matrix_mult(a, b):
    n = len(a)
    m = len(b[0])
    p = len(b)
    result = [[0]*m for _ in range(n)]
    for i in range(n):
        for j in range(m):
            for k in range(p):
                result[i][j] += a[i][k] * b[k][j]
    return result